apps/labs/posts/enhancements-to-numba-guvectorize-decorator.md

```diff
@@ -0,0 +1,148 @@
+---
+title: 'Enhancements to Numba''s guvectorize decorator'
+published: February 25, 2021
+author: guilherme-leobas
+description: 'Starting from Numba 0.53, Numba will ship with an enhanced version of the @guvectorize decorator. Similar to the @vectorize decorator, @guvectorize now has two modes of operation: Eager, or decoration-time compilation and Lazy, or call-time compilation'
+category: [PyData Ecosystem]
```

Comment:
```suggestion
category: [PyData ecosystem]
```

Reply:
Doesn't show up in the filter with the E 🤷🏽‍♀️

Reply:
I'll mark `[PyData Ecosystem]` → `[PyData ecosystem]` with the lowercase 'e' in the [Labs blog migration document](https://www.notion.so/quansightlabs/Labs-blog-migration-process-200c3297b67245b3953dd23487badfe9) to catch it next time.

Is this category the only one that needs a case sensitive fix?